[2022-01-25 06:36:02,552] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-22T00:00:00+00:00 [queued]>
[2022-01-25 06:36:02,562] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-22T00:00:00+00:00 [queued]>
[2022-01-25 06:36:02,562] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:36:02,563] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 06:36:02,563] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:36:02,576] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-22 00:00:00+00:00
[2022-01-25 06:36:02,581] {standard_task_runner.py:52} INFO - Started process 3903 to run task
[2022-01-25 06:36:02,585] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-22T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpkg97uq8c', '--error-file', '/tmp/tmpf9rlatw1']
[2022-01-25 06:36:02,589] {standard_task_runner.py:77} INFO - Job 12: Subtask first_task
[2022-01-25 06:36:02,654] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-22T00:00:00+00:00 [running]> on host ceab856ace11
[2022-01-25 06:36:02,822] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-22T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-22T00:00:00+00:00
[2022-01-25 06:36:02,823] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 06:36:02,824] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 06:36:02,832] {subprocess.py:85} INFO - Output:
[2022-01-25 06:36:02,834] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 06:36:02,835] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 06:36:02,860] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220122T000000, start_date=20220125T063602, end_date=20220125T063602
[2022-01-25 06:36:02,917] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 06:36:02,948] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-25 07:27:45,685] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-22T00:00:00+00:00 [queued]>
[2022-01-25 07:27:45,697] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-22T00:00:00+00:00 [queued]>
[2022-01-25 07:27:45,697] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:45,697] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 07:27:45,697] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:45,711] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-22 00:00:00+00:00
[2022-01-25 07:27:45,716] {standard_task_runner.py:52} INFO - Started process 364 to run task
[2022-01-25 07:27:45,720] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-22T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpawznea59', '--error-file', '/tmp/tmpcr64gcxt']
[2022-01-25 07:27:45,724] {standard_task_runner.py:77} INFO - Job 12: Subtask first_task
[2022-01-25 07:27:45,895] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-22T00:00:00+00:00 [running]> on host 943a4c094194
[2022-01-25 07:27:45,949] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-22T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-22T00:00:00+00:00
[2022-01-25 07:27:45,950] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 07:27:45,950] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 07:27:45,958] {subprocess.py:85} INFO - Output:
[2022-01-25 07:27:45,960] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 07:27:45,961] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 07:27:45,986] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220122T000000, start_date=20220125T072745, end_date=20220125T072745
[2022-01-25 07:27:46,012] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 07:27:46,041] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
